pandas/core/frame.py

```diff
@@ -883,8 +883,7 @@ def iterrows(self):
 
     def itertuples(self, index=True, name="Pandas"):
         """
-        Iterate over DataFrame rows as namedtuples, with index value as first
-        element of the tuple.
+        Iterate over DataFrame rows as namedtuples.
```

Comment:
Not sure if it was so important. But if you removed the comment about the index just to make it fit in one line, you can have a description as long as you want after this single-line (just separate them with a blank line.

Reply:
I noticed that in the Parameters section it mentioned that the return of the index was the first element of the tuple when `index=True` so I removed it in the description. If there is extra description that is needed I can definitely add it!